opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/auto_instrumentation.py

```diff
@@ -15,22 +15,33 @@
 # limitations under the License.
 
 from logging import getLogger
-from runpy import run_path
+from os import environ, execl
+from os.path import abspath, dirname, pathsep
+from shutil import which
 from sys import argv
 
-from pkg_resources import iter_entry_points
-
 logger = getLogger(__file__)
 
 
 def run() -> None:
 
-    for entry_point in iter_entry_points("opentelemetry_instrumentor"):
-        try:
-            entry_point.load()().instrument()  # type: ignore
-            logger.debug("Instrumented %s", entry_point.name)
+    python_path = environ.get("PYTHONPATH", [])
+
+    if not python_path:
+        python_path = []
+
```

Comment:
`else`?

Reply:
sorry, `else` what?

Reply:
You have the following:
```
    if not python_path:
        python_path = []

    if python_path:
        python_path = python_path.split(pathsep)
```

Could be converted to 

```
    if not python_path:
        python_path = []
    else:
        python_path = python_path.split(pathsep)
```

Reply:
Yes, you are right, fixing. :sweat_smile: